### ISSUE TITLE
Enable Video Playback

### ISSUE THREAD

**Issue by drewjex** [drewjex/Grid-Playground#5]
Build functionality to add videos to canvas that automatically play/stream content.

**Comment by jextrevor** [comment]
We've successfully added a video that plays, is this referring to actually uploading a new video to the canvas?

**Comment by drewjex** [comment]
Yes, that would be the idea. Or creating a video that shows your webcam so you can do live streaming or whatever.

**Comment by drewjex** [comment]
Or perhaps paste a youtube video or something.
